src/main/java/ch/njol/skript/lang/SkriptEventInfo.java

```diff
@@ -27,38 +27,30 @@
 import ch.njol.skript.SkriptAPIException;
 
 public final class SkriptEventInfo<E extends SkriptEvent> extends SyntaxElementInfo<E> {
-	
-	public Class<? extends Event>[] events;
-	public final String name;
-	
-	private final String id;
-	
-	@Nullable
-	private String[] description;
-	@Nullable
-	private String[] examples;
-	@Nullable
-	private String since;
+
 	@Nullable
-	private String documentationID;
+	private String[] description, examples, requiredPlugins;
+
 	@Nullable
-	private String[] requiredPlugins;
-	
+	private String since, documentationID;
+
+	public Class<? extends Event>[] events;
+	private final String name, id;
+
 	/**
 	 * @param name Capitalised name of the event without leading "On" which is added automatically (Start the name with an asterisk to prevent this).
 	 * @param patterns
 	 * @param c The SkriptEvent's class
 	 * @param originClassPath The class path for the origin of this event.
 	 * @param events The Bukkit-Events this SkriptEvent listens to
 	 */
-	public SkriptEventInfo(String name, final String[] patterns, final Class<E> c, final String originClassPath, final Class<? extends Event>[] events) {
+	public SkriptEventInfo(String name, String[] patterns, Class<E> c, String originClassPath, Class<? extends Event>[] events) {
```

Comment:
Change `c` to `type`, `eventType`, or just anything that isn't a single letter

Reply:
c is for class and skript uses this everywhere

Reply:
Java gave us more than one letter to define variable names for good reason, we should probably use it.